mvp/resources/README

```diff
@@ -1 +1,71 @@
+
+The URL to view the locally hosted documentation is returned: http://localhost:9080/
+
+
+Running the Galasa website on an internal server
+----------------------------------------------------
+ 
+To host the website on an internal server so that it can be accessed by other users, set the host 
+system properties or environment variables to bind to an externally available network interface, 
+rather than localhost. For example: 
+
+`java -Dserver.http.port=12345 -jar docs.jar` or `SERVER_HTTP_HOST=example.com java -jar docs.jar`
+
+ 
+Getting started 
+--------------------                                                                                                                 
+                                                                                                                                                   
+For information about installing the Galasa plug-in and getting started with using Galasa, go 
+to _Docs > Installing the Galasa plug-in_ documentation on the Galasa website.        
+
+Note: the locally run website currently links to the external Javadoc site. You can access the Javadoc 
+locally by using the Javadoc documentation that is contained in the Javadoc directory provided in this zip. 
+                                                                                                                                                    
+                                                                                                                    
+IBM Knowledge Centre
+-----------------------                                                                                                                                
+                                                                                                                           
+For information about using IBM Z Virtual Test Platform and IBM Distribution for Galasa to automate your testing, see the IBM Z Virtual Test Platform documentation at:                      
+https://www.ibm.com/docs/zvtp                                                                                                                       
+                                                                                                                                                    
+                                                                                                                                                    
+*******************************************************************************     
+
+(C) Copyright IBM Corporation 2021 All rights reserved.                                                                                             
+    For legal information, see http://www.ibm.com/legal/copytrade.shtml                                                                             
+*******************************************************************************                                                                     
```

Comment:
have we checked with Paul Taylor that he is happy with this copyright statement?

Reply:
I checked a template for a readme on the mainframe downloads site and used that. Paul is reviewing if it is needed.

Reply:
Updated to (c) Copyright IBM Corp. 2021 and dropped URL